Examples/README.md

```diff
@@ -1,12 +1,12 @@
-ITK Examples Directory
-----------------------
+ITK Sphinx Examples Directory
+-----------------------------
```

Comment:
The examples in this folder do not correspond to the Sphinx examples, but are intended to provide the examples for the SW Guide so I would not change the title. @thewtex probably revert this change?

Reply:
@jhlegarreta good catch, thanks!

Overall, this naming change will provide more differentiation and help avoid confusions with the sphinx examples / software guide examples.